docker-compose.yml

```diff
@@ -117,6 +127,8 @@ services:
       - RABBITMQ_DEFAULT_USER=olympia
       - RABBITMQ_DEFAULT_PASS=olympia
       - RABBITMQ_DEFAULT_VHOST=olympia
+    volumes:
+      - data_rabbitmq:/var/lib/rabbitmq
```

Comment:
As mentioned in my other comment I don't think we need to persist this to a volume (rabbitmq is only used as a celery broker, it's ok to lose any pending jobs locally when you mess around with your containers IMHO). In fact it's more annoying if we do, cause if we need to trash only rabbitmq data I'm not sure how to do it easily...

Reply:
See above.